README.md

```diff
@@ -70,6 +70,13 @@ AIPscan report.
 
 ![screencap5](screencaps/aipscan_hello_world.png)
 
+### Typesense integration
+
+* Install/run Typesense
+* `TYPESENSE_API_KEY="xOxOxOxO" ./tools/index-refresh  # Index AIPscan data`
+* `TYPESENSE_API_KEY="xOxOxOxO" ./tools/index-summary  # Show index summary`
+* `TYPESENSE_API_KEY="xOxOxOxO" python run.py  # Sepecify Typesense API key`
+
```

Comment:
It would be great to include it in the Docker Compose environment by default. And maybe add a bit more info here on how to install/run Typesense and how to configure it in AIPscan.

Reply:
Done!

Reply:
Neat, thanks!